e2e-tests/spec/create-survey.spec.ts

```diff
@@ -45,15 +45,45 @@ describe('create survey flow', () => {
   });
 
   it('adds a new survey', async () => {
-    await helper.addNewSurvey();
-    await helper.setSurveyMetadata(
-      SURVEY_TITLE,
-      SURVEY_DESCRIPTION,
-      JOB_NAME,
-      AD_HOC
-    );
-    await helper.addAllTasks(AD_HOC ? LOI_TASK_TYPE : null);
-    await helper.shareSurvey(USER);
-    await helper.verifySurveyCreated();
+    try {
+      await helper.addNewSurvey();
+    } catch (e) {
+      console.error(e);
+      console.log((e as Error).stack);
+      throw new Error(`Unable to addNewSurvey: ${(e as Error).message}`);
+    }
+    try {
+      await helper.setSurveyMetadata(
+        SURVEY_TITLE,
+        SURVEY_DESCRIPTION,
+        JOB_NAME,
+        AD_HOC
+      );
+    } catch (e) {
+      console.error(e);
+      console.log((e as Error).stack);
+      throw new Error(`Unable to setSurveyMetadata: ${(e as Error).message}`);
+    }
+    try {
+      await helper.addAllTasks(AD_HOC ? LOI_TASK_TYPE : null);
```

Comment:
This method might be more readable if try/catch blocks are inside each helper.

Reply:
Good call, done.